.travis.yml

```diff
@@ -25,19 +25,10 @@ install:
     - if [ "$CHECKS" = "yes" ]; then travis_retry composer require --no-interaction fabpot/php-cs-fixer:^1.11; fi;
```

Comment:
yes. And phpunit too, since we don't need php-invoker (another comment of yours).
Travis have it installed by default too, so it's so great :D